proxylib/proxylib_memcached_test.go

```diff
@@ -572,6 +572,23 @@ var textTestCases = []testCase{
 			}, proxylib.OK, "")
 		},
 	},
+	{
+		"text set pass on empty rule",
+		`		        rule: <
+		          key: "keyExact"
+		          value: ""
+				>
```

Comment:
You should make the rule really empty ("").

Reply:
Also, does it make sense to allow empty string as the "keyExact" value? Or is it the same as no key?

Reply:
done